x/axelarnet/types/msg_add_chain.go

```diff
@@ -55,10 +55,18 @@ func (m AddCosmosBasedChainRequest) ValidateBasic() error {
 		return sdkerrors.Wrap(err, "invalid address prefix")
 	}
 
+	if len(m.NativeAssets) == 0 {
```

Comment:
Is a Cosmos chain allowed to not have a native token?

Reply:
I think you can add them after the fact, with `register-asset`

Reply:
@haiyizxx can confirm, but I believe every chain is allowed to not have a native asset. Not having a native asset just means we are not supporting it.

Reply:
Cool, remove the check